Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+@Library('linkurious-shared')_
```

Comment:
@enys what does this mean?
rest-client doesn't depend on shared and shared is a private repo :)

Reply:
https://www.jenkins.io/doc/book/pipeline/shared-libraries/

Reply:
calling it linkurious-shared is going to confuse everyone, can we rename it to linkurious-jenkins-shared? (or linkurious-jenkins-common)

Reply:
I don't think we should prefix, in a specific context. ie would it make sense to call linkurious-shared linkurious-shared-node ? Here you are specificaly in a Jenkinsfile, It does not look at all like a nodeJs import.